src/components/pages/order/navbar/NavbarRightSide.js

```diff
@@ -1,11 +1,41 @@
-import { Link } from "react-router-dom";
+
 import styled from "styled-components";
+import ToggleButton from "../../../reusable-ui/ToggleButton";
 import Profile from "./Profile";
+import { useState } from "react"
+import ToastAdmin from "./ToastAdmin";
+import { toast } from "react-toastify";
+
 
 export default function NavbarRigthSide({username}) {
+
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+
+  const displayToastNotification = () => {
+    if (!isModeAdmin) {
+      toast.info("Mode admin activé", {
+        // icon: <FaUserSecret size={30} />,
+        theme: "dark",
+        position: "bottom-right",
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      })
+    }
+    setIsModeAdmin(!isModeAdmin)
```

Comment:
```suggestion
    setIsAdmin(!isAdmin)
```
( si renaming validé par Vi )